components/automate-ui/src/app/components/process-progress-bar/process-progress-bar.component.ts

```diff
@@ -38,7 +38,7 @@ export class ProcessProgressBarComponent implements OnInit {
         }
         this.applyRulesInProgress = state === ApplyRulesStatusState.Running;
         if (!this.cancelRulesInProgress && state === ApplyRulesStatusState.Running) {
-          this.percentageComplete = percentageComplete;
+          this.percentageComplete = Math.floor(percentageComplete * 100);
```

Comment:
This will work, in that it completes the task of all accesses to "percentageComplete" handling it the same way, multiplying by 100. However, we could alternately make the fix more robust by changing the source in the UI rather than each use of it in the UI.

That would be changing this line:
https://github.com/chef/automate/blob/8afdaf15b72920fe5af046b18ee3ebe25f194f13/components/automate-ui/src/app/entities/projects/project.reducer.ts#L129
to this:
```
      return set(
        APPLY_RULES_STATUS,
        mapKeys(key => camelCase(key),
          {
            ...action.payload,
            percentage_complete: action.payload.percentage_complete * 100
          }),
        state);
```

Reply:
okay, i will try.. thank you!